crest/Assets/Crest/Crest-Examples/Main/Data/SettingsFoam.asset

```diff
@@ -17,4 +17,4 @@ MonoBehaviour:
   _waveFoamCoverage: 0.8
   _shorelineFoamMaxDepth: 0.8
   _shorelineFoamStrength: 0.5
-  _renderTextureFormat: 15
+  _renderTextureFormat: 45
```

Comment:
That's correct right? I assume so but not easy to tell :). Are there any other bits of data like this that need fixup?

Reply:
Yes and no.

I could use _RenderTextureFormat_ for the _SimSettingsFoam_ and use _GraphicsFormatUtility.GetGraphicsFormat_ to convert it in the _LodDataMgrFoam_? By doing so users get the much simpler _RenderTextureFormat_ enum dropdown (28 options versus 100+ options).

It has the added benefit of not having to change the serialised enum value. Otherwise, I think we should change the name of this property so it reverts to a sane default instead of having the wrong format serialised.

Reply:
Thanks, i think i get it. Just realised if people have existing foam data, then it could give them the wrong value now if we use GraphicsFormat? Shall we just make a clean break and rename this param to _renderTextureGraphicsFormat? That will still lose the values users have currently (if anyone has actually changed it - prob unlikely), but it wont suddenly give them a completely different value? Lmk if that makes sense, i may not be understanding.

Reply:
Sounds good. Done.